roles/identity-management/manage-aws-identities/README.md

```diff
@@ -31,6 +31,18 @@ identities:
 
   profiles:
     - profile_name: aws-profile-na-1
+      policies:
+        - policy_name: CustomManagedPolicy
+          policy_description: "Describe your custom managed policy"
+          policy:
```

Comment:
As discussed offline, let's pull this up one level to allow for independent management of policies; Same goes for groups below.